### PR TITLE
Override method result() for catch RuntimeExcption

### PR DESCRIPTION
**Pull request by imixac**
#6

### CODE REVIEW

File: src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java [hunk 1]
```diff
@@ -16,4 +18,27 @@ public String getStepName(Step step) {
                 : step.getKeyword() + step.getName();
 
     }
+
+//    @Override
+//    Добавил обработку RuntimeException
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java]: Only English plz

File: src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java [hunk 2]
```diff
@@ -16,4 +18,27 @@ public String getStepName(Step step) {
                 : step.getKeyword() + step.getName();
 
     }
+
+//    @Override
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java]: Why is Override commented out?

File: src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java [hunk 3]
```diff
@@ -16,4 +18,27 @@ public String getStepName(Step step) {
...
+//    Добавил обработку RuntimeException
+//    Но мне кажется это бред, потому что private нельзя оверайдить
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java]: Use javadoc capable comment for method

File: src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java [hunk 4]
```diff
@@ -16,4 +18,27 @@ public String getStepName(Step step) {
...
+//        if (callback != null){
+//            try {
+//                callbackResult = callback.newInstance().call();
+//            } catch (RuntimeException | ReflectiveOperationException ex){
+//                throw new AllureException("Could not initialize callback", ex);
+//            }
+//        }
+//    }
+
+//    Саму ошибку решил тут обрабатывать
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java]: Remove this comment

File: src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java (hunk 3 above)
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java]: Agree.

**Review by clicman — CHANGES_REQUESTED** [review]

File: src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java [hunk 5]
```diff
@@ -16,4 +17,9 @@ public String getStepName(Step step) {
                 : step.getKeyword() + step.getName();
 
     }
+
+    @Override
+    public Throwable getError(Result result){
+                return result.getError().getCause();
```
Comment (kosteman) [inline src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java]: too many tabs

File: src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java [hunk 6]
```diff
@@ -16,4 +17,9 @@ public String getStepName(Step step) {
                 : step.getKeyword() + step.getName();
 
     }
+
+    @Override
+    public Throwable getError(Result result){
```
Comment (kosteman) [inline src/main/java/ru/sbtqa/tag/allure/TagAllureReporter.java]: add space between ) and {

**Comment by imixac** [comment]
@kosteman allure-cucumberjvm merged
check pull request

**Comment by clicman** [comment]
We should wait for release.

чт, 21 сент. 2017 г. в 18:24, imixac <notifications@github.com>:

**Comment by clicman** [comment]
http://central.maven.org/maven2/ru/yandex/qatools/allure/allure-cucumber-jvm-adaptor/1.6.4/
Released

**Comment by clicman** [comment]
@imixac  update pom.xml to 1.6.4

**Pull request merged**